src/Native/Runtime/CMakeLists.txt

```diff
@@ -166,6 +173,9 @@ else()
 
   list(APPEND COMMON_RUNTIME_SOURCES
     unix/PalRedhawkUnix.cpp
+    ../gc/unix/gcenv.unix.cpp
```

Comment:
I have noticed that gcenv.unix.cpp has several TODOs.

In particular, `GCToOSInterface::GetCacheSizePerLogicalCpu` is important for a good GC performance. It was not implemented for CoreRT before your change either, so it is not a regression. I would be nice to get it fixed in future change.